connectivity/FEATURE_BLE/include/ble/services/BatteryService.h

```diff
@@ -16,6 +16,8 @@
  * limitations under the License.
  */
 
+#warning "These services are deprecated and will be removed. Please see services.md for details about replacement services."
```

Comment:
lets use here `MBED_DEPRECATED`, so when we are about to remove them we use search and remove